std-bits/microsoft/src/main/java/org/enso/microsoft/azure/AzureCredential.java

```diff
@@ -34,32 +34,14 @@ record ClientSecret(HideableValue tenantId, HideableValue clientId, HideableValu
       implements AzureCredential {
     @Override
     public String uniqueId() {
-      return "ClientSecret("
-          + makeUnique(tenantId)
-          + ", "
-          + makeUnique(clientId)
-          + ", "
-          + makeUnique(clientSecret)
-          + ")";
+      return "ClientSecret(" + tenantId + ", " + clientId + ", " + clientSecret + ")";
     }
   }
 
   record BlobStorageSASToken(HideableValue token) implements AzureCredential {
     @Override
     public String uniqueId() {
-      return "BlobStorageSASToken(" + makeUnique(token) + ")";
+      return "BlobStorageSASToken(" + token + ")";
     }
   }
-
-  private static String makeUnique(HideableValue value) {
```

Comment:
- this "decomposition" is violating the new encapsulation
- now, the `makeUnique` operation has to be done in the `HdeableValue` or impl.
- I was considering to add new `String makeUnique()` method
- but then I realized I can just reuse `toString()` and it will all be even simpler